### DISCUSSION TITLE
Welcome to sigaa_ufc_android Discussions!

### DISCUSSION THREAD

**Discussion by rodrigmatrix** [rodrigmatrix/sigaa_ufc_android#38] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by apollotsantos** [comment]
Olá @rodrigmatrix , como vai?

Sou aluno da UFSJ, e começamos a usar o SIGAA agora. Queria utilizar seu app como base para disponibilizar aos alunos, se você permitir. 

Fazendo uns testes, obtive erro no login mesmo mudando as URLs no source. Poderia me dar uma assistência depois?

Sds,

Thiago Santos

PS: Não sei se recebeu meu email. Eles estão indo pro spam. Vou tentar por aqui e pelo git por segurança kkk

**Reply by krymancer** [reply]
Olá @apollotsantos já verificou como os cookies se comportam? as implementações do sigga podem mudar de acordo de como ele é hospedado pela sua universidade, recomendo tentar utilizar algum scrapper se não estiver familiarizado com a técnica utilizada nesse app. Pode usar algo como python e tentar pegar a sessão e replicar no scrapper que tem no projeto. espero que ajude

**Reply by apollotsantos** [reply]
Olá @Krymancer, por acaso estava mexendo no projeto nesse momento! Já inspecionei o tráfego com um proxy e parece que o cookie fica da mesma forma. Conferi URL por URL também. O erro que tenho após o login é o seguinte

`D/FA: Connected to remote service
V/FA: Processing queued up service tasks: 1
W/View: couldn't find view with id 2131296806
W/View: couldn't find view with id 2131296806`

Te diz algo? Tenho experiencia em programação backend, mas estou aprendendo android.

**Reply by apollotsantos** [reply]
Adicionalmente, esse erro só aparece quando as credenciais estão corretas. Caso eu entre com a credencial errada ele acusa que a combinação de usuário e senha está incorreta.
